git_machete/cmd.py

```diff
@@ -1470,181 +1469,186 @@ def check_required_param(in_args, allowed_values):
-        read_definition_file()
-        delete_unmanaged()
-    elif cmd in ("d", "diff"):
-        param = check_optional_param(parse_options(args, "s", ["stat"]))
-        # No need to read definition file.
-        diff(param) # passing None if not specified
-    elif cmd == "discover":
-        expect_no_param(parse_options(args, "l", ["list-commits"]))
-        # No need to read definition file.
-        discover_tree()
-    elif cmd in ("e", "edit"):
-        expect_no_param(parse_options(args))
-        # No need to read definition file.
-        edit()
-    elif cmd == "file":
-        expect_no_param(parse_options(args))
-        # No need to read definition file.
-        print definition_file
-    elif cmd == "fork-point":
-        param = check_optional_param(parse_options(args))
-        # No need to read definition file.
-        print fork_point(param or current_branch())
-    elif cmd == "format":
-        # No need to read definition file.
-        usage("format")
-    elif cmd in ("g", "go"):
-        param = check_required_param(parse_options(args), directions)
-        read_definition_file()
-        cb = current_branch()
-        dest = parse_direction(cb, down_pick_mode=True)
-        if dest != cb:
-            go(dest)
-    elif cmd == "help":
-        param = check_optional_param(parse_options(args))
-        # No need to read definition file.
-        usage(param)
-    elif cmd == "infer":
-        expect_no_param(parse_options(args, "l", ["list-commits"]))
-        # No need to read definition file.
-        discover_tree()
-    elif cmd == "list":
-        allowed_values = "managed|slidable|slidable-after <branch>|unmanaged"
-        in_args = parse_options(args)
-        if not in_args or len(in_args) > 2:
-            raise MacheteException("'%s' expects argument(s): %s" % (cmd, allowed_values))
-        elif not in_args[0]:
-            raise MacheteException("Argument to '%s' cannot be empty; expected %s" % (cmd, allowed_values))
-        elif in_args[0][0] == "-":
-            raise MacheteException("option '%s' not recognized" % in_args[0])
-        elif in_args[0] in ("managed", "slidable", "unmanaged") and len(in_args) == 2:
-            raise MacheteException("'%s %s' doesn't expect an extra argument" % (cmd, in_args[0]))
-        elif in_args[0] == "slidable-after" and len(in_args) == 1:
-            raise MacheteException("'%s %s' requires an extra <branch> argument" % (cmd, in_args[0]))
-
-        param = in_args[0]
-        read_definition_file()
-        if param == "managed":
-            res = managed_branches
-        elif param == "slidable":
-            res = slidable()
-        elif param == "slidable-after":
-            b = in_args[1]
-            expect_in_managed_branches(b)
-            res = slidable_after(b)
-        elif param == "unmanaged":
-            res = excluding(local_branches(), managed_branches)
+def main():
+    try:
+        cmd = None
+        opt_debug = False  # noqa
```

Comment:
Pls reapply my changes you commited before (global vars etc.)

Reply:
That commit https://github.com/VirtusLab/git-machete/commit/7c686f03a23b0c29030152949eb18ff79e3bf61e ? Is there a PR for it or it is included in this PR?

Reply:
It used to be included in this PR before you apparently squashed the commits :/

Reply:
I omit it, not squashed. I will include it back.

Reply:
I am including back https://github.com/VirtusLab/git-machete/commit/7c686f03a23b0c29030152949eb18ff79e3bf61e